tests/test_annotations.py

```diff
@@ -31,14 +31,16 @@ class C:
             y = attr.ib(type=str)
             z = attr.ib()
 
+        attr.resolve_types(C)
+
         assert int is attr.fields(C).x.type
         assert str is attr.fields(C).y.type
         assert None is attr.fields(C).z.type
-        assert C.__init__.__annotations__ == {
+        assert {
             "x": int,
             "y": str,
-            "return": None,
-        }
+            "return": type(None),
```

Comment:
I would probably define a constant called `NoneType = type(None)`. Or I might define a helper that tests this. 

```
def assert_init_annotation(cls, **annotations):
    annotations["return"] = type(None)
    assert annotations == typing.get_type_hints(cls.__init__)

assert_init_annotation(C, x=int, y=str)
```

Reply:
Excellent idea!